beancount/parser/hashsrc.py

```diff
@@ -18,6 +18,7 @@
 PARSER_SOURCE_FILES = [
     'lexer.l',
     'grammar.y',
+    'macros.h',
```

Comment:
Do we need to add anything to setup.py?

Reply:
If you mean to include the new file in the distribution, I don't think so: we use a wildcard expansion in MANIFEST.in